pkg/services/authn/authnimpl/sync/rbac_sync.go

```diff
@@ -97,10 +96,12 @@ func (s *RBACSync) fetchPermissions(ctx context.Context, ident *authn.Identity)
 	return permissions, nil
 }
 
-var fixedCloudRoles = map[org.RoleType]string{
-	org.RoleViewer: accesscontrol.FixedCloudViewerRole,
-	org.RoleEditor: accesscontrol.FixedCloudEditorRole,
-	org.RoleAdmin:  accesscontrol.FixedCloudAdminRole,
+// Since Cloud Admin/Editor/Viewer roles are not yet implemented one-to-one in the Grafana, it becomes a confusing experience for users,
+// therefore we are doing granular mapping of all available functionality in the Grafana temporary.
+var fixedCloudRoles = map[org.RoleType][]string{
+	org.RoleViewer: {accesscontrol.FixedCloudViewerRole, accesscontrol.FixedCloudSupportTicketReader},
+	org.RoleEditor: {accesscontrol.FixedCloudEditorRole, accesscontrol.FixedCloudSupportTicketAdmin},
```

Comment:
Mapping an editor to `FixedCloudSupportTicketAdmin` as it seems now Cloud Editors can create, comment and read support tickets.